applicationinsights-react-js/test/AppInsightsErrorBoundary.test.tsx

```diff
@@ -85,4 +92,76 @@ describe("<AppInsightsErrorBoundary />", () => {
       }
     }
   });
+
+  function BackButton() {
+    const navigate = useNavigate();
+    console.log("BackButton", BackButton);
+    return (
+      <button type="button" onClick={() => navigate(-1)}>Back</button>
+    );
+  }
+
+  function NewError() {
+    return (
+      <div>
+        <BackButton />
+        <div>You are on the error page</div>
+      </div>
+    );
+  }
+
+  test("error state would not show again after user use back button", async () => {
+    const orgError = console && console.error;
+    if (orgError) {
+      console.error = msg => { /* Do Nothing */ };
+    }
+    const Home = () => <div>Home Page</div>;
+    const About = () => {
+      throw new Error("something went wrong");
+    };
+    const NewPage = () => <div>New Page</div>;
+
+    try {
+      render(
+        <AppInsightsErrorBoundary appInsights={reactPlugin} onError={NewError}>
+          <Router>
+            <div>
+              <Link to="/">Home</Link>
+              <Link to="/about">About</Link>
+              <Link to="/newpage">newPage</Link>
+              <BackButton />
+              <Routes>
+                <Route path="/" element={<Home />} />
+                <Route path="/about" element={<About />} />
+                <Route path="/newpage" element={<NewPage />} />
+              </Routes>
+              <LocationDisplay />
+            </div>
+          </Router>
+        </AppInsightsErrorBoundary>
+      );
+      expect(screen.getByText(/Home Page/i)).toBeInTheDocument()
+
+       // navigate to new page 
+       await userEvent.click(screen.getByText(/newpage/i));
+       // go back to home page (no error, so show home component)
+      await userEvent.click(screen.getByText(/Back/i));
+      expect(screen.getByText(/Home Page/i)).toBeInTheDocument();
+      
+      
+      // navigate to about page (throws error, so show error component)
+      await userEvent.click(screen.getByText(/about/i)); // compile error useNavigate() may be used only in the context of a <Router> component.
```

Comment:
for the navigate(-1), it could be used to switch back from NewPage to homePage. But as the about page throws an error and rendered the errorPage outside router, navigate is no longer supported. So this unit test failed to reproduce the bug.